src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
variables must be created so that the next programmer who enters the project can see what is in this variable.
We must specify what is in our variable more correctly(In all the cases)
```suggestion
  const listMaleGender = century
```